packages/backend/src/graphql/schema.graphql

```diff
@@ -2,6 +2,17 @@ type Query {
   account(id: String!): Account
 }
 
+type PageInfo {
+  "Paginating forwards: the cursor to continue."
+  endCursor: String
+  "Paginating forwards: Are there more pages?"
+  hasNextPage: Boolean!
+  "Paginating backwards: Are there more pages?"
+  hasPreviousPage: Boolean!
+  "Paginating backwards: the cursor to continue."
+  startCursor: String
+}
+
 type Amount {
   amount: Int!
```

Comment:
We should add an issue to update all the amount usages from Int to the UInt64?

Reply:
removed as unnused